#!/Users/chloet/Documents/pdlpy/test/benchmarks/env/e5335ded34b2270c0abe8b1c6a82d4ad/bin/python3.9
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
